Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.cs

```diff
@@ -143,5 +146,19 @@ private void AdjustBladeItemSize()
                 }
             }
         }
+
+        private void ItemsVectorChanged(IObservableVector<object> sender, IVectorChangedEventArgs e)
+        {
```

Comment:
Check `BladeMode == BladeMode.Fullscreen` at the top and exit if not FullScreen. No need to cast sender and check the hash when not full screen